src/main/java/io/unlogged/logging/util/NetworkClient.java

```diff
@@ -71,28 +73,30 @@ public void sendPOSTRequest(String url, String attachmentFilePath) throws IOExce
         try {
             form = new MultipartUtility(url, charset, headers);
 
-            File binaryFile = new File(attachmentFilePath);
-            form.addFilePart("file", binaryFile);
-            form.addFormField("sessionId", sessionId);
-            form.addFormField("hostname", getHostname());
-
+			List<File> listUpload = new ArrayList<>();
+			listUpload.add(new File(attachmentFilePath));
+			listUpload.add(new File(loggerPath));
+            form.addFilePart("file", listUpload);
```

Comment:
why not 
```java
.addFilePart("file", 
.addFilePart("logFile", 
```

and then you wouldn't require the change in `MultipartUtility` class ?

Reply:
yeah, the is a better approach for both the places. Done.